samples/cpp/whisper_speech_recognition/README.md

```diff
@@ -0,0 +1,38 @@
+# Whisper automatic speech recognition sample
+
+This example showcases inference of speech recognition Whisper Models. The application doesn't have many configuration options to encourage the reader to explore and modify the source code. For example, change the device for inference to GPU. The sample fearures `ov::genai::WhisperPipeline` and uses audio file in wav format as an input source.
+
+## Download and convert the model and tokenizers
+
+The `--upgrade-strategy eager` option is needed to ensure `optimum-intel` is upgraded to the latest version.
+
+It's not required to install [../../requirements.txt](../../requirements.txt) for deployment if the model has already been exported.
+
+```sh
+pip install --upgrade-strategy eager -r ../../requirements.txt
+optimum-cli export openvino --trust-remote-code --model openai/whisper-base whisper-base
+```
+
+## Prepare audio file
+
+Prepare audio file in wav format with sampling rate 16k Hz.
+
+## Run
+
+`whisper_speech_recognition whisper-base sample.wav`
+
+
+Discrete GPUs (dGPUs) usually provide better performance compared to CPUs. It is recommended to run larger models on a dGPU with 32GB+ RAM. For example, the model meta-llama/Llama-2-13b-chat-hf can benefit from being run on a dGPU. Modify the source code to change the device for inference to the GPU.
+
+Models can be downloaded from [OpenAI HiggingFace](https://huggingface.co/openai).
```

Comment:
could you please mention models which you have tried and they are compatible with this sample?

Reply:
I added models list. I also added todo to move it from samples to supported models md.